apps/model_eval_viz/app.py

```diff
@@ -83,58 +84,64 @@ def _get_data(_filepath:str)->gpd.GeoDataFrame:
     title='United States Streamgage Map'
 )
 
-# Instantiate template
-
 # Plotting and Servable execution 
 stream_gage = _get_data(path)
 
 # Widget setup to select multiple states
 state_list = list(states['shapeName'].unique())
 #sort alphabetically
 state_list.sort()
+
+streamgage_input = pn.widgets.TextInput(
+    name='Streamgage Site ID', 
+    placeholder='Streamgage Site ID #',
+    
+    )
+
 state_selector = pn.widgets.MultiSelect(
     description="Hold ctrl to toggle multiple states",
     name="Select a state",
     options=state_list,
 )
 
-
 base_map_options = {
     'OpenStreetMap': gv.tile_sources.OSM,
     'ESRI Imagery': gv.tile_sources.EsriImagery,
     'ESRI World Street Map': gv.tile_sources.EsriWorldStreetMap,
 }
 
+subset = ['nldi','swim','gfv1d1','camels']
```

Comment:
Let's add this constant to the config.py and called it something like STREAMGAGE_SUBSET